pkg/endpoint/endpoint.go

```diff
@@ -1391,6 +1391,15 @@ func (e *Endpoint) GetK8sPodName() string {
 	return k8sPodName
 }
 
+// HumanStringLocked returns the endpoint's most human readable identifier as string
+func (e *Endpoint) HumanStringLocked() string {
+	if pod := e.GetK8sNamespaceAndPodNameLocked(); pod != "" {
```

Comment:
When writing the CEP controller I noticed cases where this would return empty strings for some time. If that happens, our string will be wrong in `(d *Daemon) allocateIPsLocked`, right? Obviously you didn't see it when testing so I'm not sure if it still happens for us.

Reply:
It's only used on restore and k8s pod name is serialized so it will be available and otherwise we all-back.